interview-questions.md

```diff
@@ -8,62 +8,65 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
+  Your answer: In my experience parameters are parenthesis that will hold arguments. arguments can be used to get a certain output from your code.
 
-  Researched answer:
+  Researched answer: A parameter is a variable that is passed into a function. While argument hold a value inside a function.
```

Comment:
The parameter is a placeholder within the function, while the argument is the data being passed in when invoking a function.